In [10]:
with open('../../code/2/popular-names.txt') as f:
    content = f.readlines()
length = len(content)
dic = {}

for line in content:
    name = line.split("\t")[0]

    if name not in dic:
        dic[name] = 1
    else:
        dic[name] +=1 

for name in sorted(dic):
    print(dic[name],name)

17 Abigail
3 Aiden
8 Alexander
8 Alexis
10 Alice
20 Amanda
2 Amelia
14 Amy
21 Andrew
15 Angela
41 Anna
5 Annie
6 Anthony
23 Ashley
4 Austin
14 Ava
32 Barbara
4 Benjamin
12 Bertha
2 Bessie
24 Betty
7 Brandon
13 Brian
10 Brittany
15 Carol
1 Carolyn
75 Charles
5 Charlotte
4 Chloe
43 Christopher
8 Clara
1 Crystal
11 Cynthia
31 Daniel
57 David
13 Deborah
9 Debra
20 Donald
12 Donna
10 Doris
36 Dorothy
40 Edward
3 Elijah
73 Elizabeth
26 Emily
35 Emma
15 Ethan
16 Ethel
3 Evelyn
20 Florence
15 Frances
43 Frank
5 Gary
58 George
13 Hannah
3 Harper
7 Harry
15 Heather
45 Helen
23 Henry
8 Ida
15 Isabella
25 Jacob
118 James
13 Jason
5 Jayden
5 Jeffrey
26 Jennifer
25 Jessica
9 Joan
108 John
70 Joseph
31 Joshua
8 Judith
1 Julie
4 Justin
18 Karen
4 Kathleen
1 Kelly
14 Kimberly
2 Larry
1 Laura
2 Lauren
7 Liam
4 Lillian
26 Linda
18 Lisa
2 Logan
1 Lori
1 Lucas
18 Madison
60 Margaret
20 Marie
16 Mark
92 Mary
8 Mason
37 Matthew
3 Megan
18 Melissa
10 Mia
74 Michael
15 Michelle
24 Mildred
12 Minnie
22 Nancy
10

UNIXコマンド
cut -f 1  'code/2/popular-names.txt' | sort| uniq -c

実行結果
 17 Abigail
   3 Aiden
   8 Alexander
   8 Alexis
  10 Alice
  20 Amanda
   2 Amelia
  14 Amy
  21 Andrew
  15 Angela
  41 Anna
   5 Annie
   6 Anthony
  23 Ashley
   4 Austin
  14 Ava
  32 Barbara
   4 Benjamin
  12 Bertha
   2 Bessie
  24 Betty
   7 Brandon
  13 Brian
  10 Brittany
  15 Carol
   1 Carolyn
  75 Charles
   5 Charlotte
   4 Chloe
  43 Christopher
   8 Clara
   1 Crystal
  11 Cynthia
  31 Daniel
  57 David
  13 Deborah
   9 Debra
  20 Donald
  12 Donna
  10 Doris
  36 Dorothy
  40 Edward
   3 Elijah
  73 Elizabeth
  26 Emily
  35 Emma
  15 Ethan
  16 Ethel
   3 Evelyn
  20 Florence
  15 Frances
  43 Frank
   5 Gary
  58 George
  13 Hannah
   3 Harper
   7 Harry
  15 Heather
  45 Helen
  23 Henry
   8 Ida
  15 Isabella
  25 Jacob
 118 James
  13 Jason
   5 Jayden
   5 Jeffrey
  26 Jennifer
  25 Jessica
   9 Joan
 108 John
  70 Joseph
  31 Joshua
   8 Judith
   1 Julie
   4 Justin
  18 Karen
   4 Kathleen
   1 Kelly
  14 Kimberly
   2 Larry
   1 Laura
   2 Lauren
   7 Liam
   4 Lillian
  26 Linda
  18 Lisa
   2 Logan
   1 Lori
   1 Lucas
  18 Madison
  60 Margaret
  20 Marie
  16 Mark
  92 Mary
   8 Mason
  37 Matthew
   3 Megan
  18 Melissa
  10 Mia
  74 Michael
  15 Michelle
  24 Mildred
  12 Minnie
  22 Nancy
  10 Nicholas
  13 Nicole
  10 Noah
   2 Oliver
  18 Olivia
   1 Pamela
  38 Patricia
   1 Rachel
   2 Rebecca
  51 Richard
 108 Robert
  10 Ronald
  39 Ruth
  19 Samantha
  16 Sandra
  26 Sarah
   1 Scott
   7 Sharon
  14 Shirley
  13 Sophia
  17 Stephanie
   7 Steven
  24 Susan
   7 Tammy
   8 Taylor
  58 Thomas
   1 Tracy
   9 Tyler
  13 Virginia
   1 Walter
 111 William